habitat/tasks/rearrange/rearrange_sim.py

```diff
@@ -261,6 +277,23 @@ def reconfigure(self, config: Config):
                 ao: ao.joint_positions for ao in self.art_objs
             }
 
+    def get_nav_pos(self, pos):
```

Comment:
All of these constants were tuned for the ReplicaCAD scenes to deal with isolated navmesh islands. Is this code still necessary even with the updated ReplicaCAD scenes? This code will fail for other assets.